docs/concepts/models/overview.md

```diff
@@ -75,11 +75,31 @@ Some SQL engines support registering comments as metadata associated with a tabl
 
 SQLMesh will automatically register comments if the engine supports it and the [connection's `register_comments` configuration](../../reference/configuration.md#connection) is `true` (`true` by default). Engines vary in their support for comments - see [tables below](#engine-comment-support).
 
-#### Model comments
+#### Model comment
 
 SQLMesh will register a comment specified before the `MODEL` DDL block as the table comment in the underlying SQL engine. If the [`MODEL` DDL `description` field](#description) is also specified, SQLMesh will register it with the engine instead.
 
-SQLMesh will automatically detect comments in a query's column selections and register each column's final comment in the underlying SQL engine.
+#### Explicit column comments
+
+You may explicitly specify column comments in the `MODEL` DDL `column_descriptions` field.
+
+Specify them as an dictionary of key/value pairs separated by an equals sign `=`, where the column name is the key and the column comment is the value. For example:
+
+```sql linenums="1" hl_lines="4-6"
+MODEL (
+  name sushi.customer_total_revenue,
+  cron '@daily',
+  column_descriptions (
+    id = 'This is the ID column comment'
+  )
+);
+```
+
+If the `column_descriptions` key is present, SQLMesh will not detect and register inline column comments from the model query.
+
+#### Inline column comments
+
+If the `column_descriptions` key is not present in the `MODEL` definition, SQLMesh will automatically detect comments in a query's column selections and register each column's final comment in the underlying SQL engine.
```

Comment:
Nit: not sure if `final` is the right term to use here. Do we use that consistently? Perhaps we can get rid of it altogether if we explain that only comments _following_ the projection (e.g., after the comma) are registered.

Reply:
I may not understand how SQLMesh behaves - what gets registered if there are two comments after the comma?

Reply:
Ah I forgot I changed this recently. 🤦

> what gets registered if there are two comments after the comma

The final one, so what you have here is actually the correct thing!